include/tvm/runtime/container.h

```diff
@@ -65,11 +66,36 @@
 #include <utility>
 #include <vector>
 
+#if defined(_MSC_VER) && !defined(__clang__)
+uint32_t __builtin_ctz(uint32_t value);
+uint32_t __builtin_clz(uint32_t value);
```

Comment:
I actually meant move all these functions to a cc file(the rehash function)

Reply:
@tqchen i see....I removed calling to those functions in the most recent update anyway